common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -293,20 +293,39 @@ object PageElement {
           case _ => None
         }).toList
 
+      case ElementType.Map => {
+        for {
+          mapElem <- element.mapTypeData
+          originalUrl <- mapElem.originalUrl
+          source <- mapElem.source
+          html <- mapElem.html
+          src <- getIframeSrc(html)
+        } yield MapBlockElement(
```

Comment:
This looks a bit like dodgy Intellij defauls here for how it indents the arguments of MapBlockElement(.... Does it pass `scalastyle`? - not sure but it should come up as part of the test task if so.